### PR TITLE
🌐 Add French definition for "plural"

### PR DESCRIPTION
**Pull request by CharlesMangwa**
Hey!

This PR aims to add the French definition for **"plural"**. 

A link has been added to the existing English definition (and vice versa).

Current French definitions (or submitted via another PR) have also been linked directly.

Thanks in advance for the proofreading! 🙏

### CODE REVIEW

File: 11ty/definitions/fr_FR/pluriel.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: Pluriel
+slug: pluriel
+speech: nom
+defined: true
+---
+
+désigne plus d'une entité vivant dans un organisme d'accueil.
```
Comment (deammer) [inline 11ty/definitions/fr_FR/pluriel.md]: Suggestion:
```
désigne plusieurs entités
```

**Review by deammer — APPROVED** [review]
👍 looks good!

File: 11ty/definitions/fr_FR/pluriel.md (hunk 1 above)
Comment (CharlesMangwa) [reply 11ty/definitions/fr_FR/pluriel.md]: Yes, has a nicer ring to it: implemented!

File: 11ty/definitions/fr_FR/pluriel.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+title: Pluriel
```
Comment (tatianamac) [inline 11ty/definitions/fr_FR/pluriel.md]:
```suggestion
title: pluriel
```
In French is it necessary to capitalise words that are not proper nouns? It's inconsistent in our English versions (which requires addressing), but curious if this should be the case in French or not.

**Review by tatianamac — APPROVED** [review]
One question to address

File: 11ty/definitions/fr_FR/pluriel.md (hunk 2 above)
Comment (deammer) [reply 11ty/definitions/fr_FR/pluriel.md]: It's not necessary, French generally follows the same rules as English in that regard 👍

Comment (tatianamac) [reply 11ty/definitions/fr_FR/pluriel.md]: Ah, thank you! Relieved to hear this as I suggested making it lowercase for all terms 🙈

Comment (CharlesMangwa) [reply 11ty/definitions/fr_FR/pluriel.md]: Yeah that one is on me, I think it's just my habit of starting everything with an uppercase. Should be fixed everywhere now, sorry!

**Comment by CharlesMangwa** [comment]
And the question has been successfully addressed: the title is now lowercase across all French definitions (unless I missed some... 🙈)

**Pull request merged**